helm/templates/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.ingress.enabled }}
```

Comment:
Add a `scope` modifier to the `gaia` namespace so it doesn't handle other namespaces.